### PR TITLE
Export: materials: support emission strength

### LINKED ISSUE #367: Exporter, does not save Emissive Factor/Emission Strength Value for materials.
Currently the Emissive Factor/Emission Strength is not exported to the .glb file.

-------
Steps to Reproduce:

Following this [example](https://docs.blender.org/manual/en/2.80/_images/addons_io-gltf2_material-emissive.png)

I set up a material using the Principal BSDF and Emission nodes.  I then make some adjustments to the Principal and Emission settings to get something looking like [this scene.](https://i.imgur.com/5Az9fEl.png) 

The nodes on each material [look like this.](https://i.imgur.com/39s8mNc.png)

After exporting an re-importing I get this. [imported nodes](https://i.imgur.com/7qaPrMR.png)

I wasn't sure if it was the import process or the export process so I checked the .glb embedded file.  It appears the emissiveFactor portion is always being saved as 1, 1, 1 (see below)

```
"emissiveFactor" : [
                1,
                1,
                1
            ]
```

Knowing that the Emissive Factor is supposed to be an RGB value I also tried attaching an RGB node and attaching it to the strength of the Emission node, however it was still not saved.

-------
Additional Notes:

I also went in and edited the "emissiveFactor" in the embedded .glb file.  Like so...
```
"emissiveFactor" : [
                0.33,
                0.33,
                0.33
            ]
```
When importing this edited file, I get a node tree [including a multiply node.](https://i.imgur.com/MfHIGt1.png) Where the Color2 of the multiply node matched my edited .glb file.  

The result looks okay, but is too bright since it still takes into account the emission node's strength.  (Which is always set to 1.0)

Upon exporting this new scene the emissiveFactor is still not saved.

-----
Using Blender 2.8 Beta Windows 64 bit, March 18th, 19:10:14 build.

Edit: updated example link

### PR DESCRIPTION
**Pull request by scurest**
~~Done on top of #1155, so do that first.~~ Rebased onto master.

Fixes #367.

Adds support for Emission Strength. The strength is just multiplied into the emissiveFactor when present.

### CODE REVIEW

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py
Comment (scurest) [inline addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py:126]: These two lines should support [D4971](https://developer.blender.org/D4971), but I haven't tested it or anything.

**Comment by julienduroure** [comment]
Ok, so this means that we can't merge this PR right now. What is the solution for now?
Export it only if blender factor is < 1.0 ? Export always, but clamp to 1.0 when necessary?

**Comment by emackey** [comment]
Export when possible, but clamped to a maximum of 1.0.  This is on the Khronos roadmap though, so hopefully someday soon we'll get an extension going that works around this limitation.

**Comment by emackey** [comment]
Actually it looks like the 1.0 clamp is already included here.

File: tests/roundtrip/11_pbr_factors/11_pbr_factors_report.json
Comment (emackey) [inline tests/roundtrip/11_pbr_factors/11_pbr_factors_report.json:2]: Once you merge or rebase master, this report file might not be needed anymore.

**Comment by scurest** [comment]
This does clamp when >1. I moved the clamp out to a separate block and put a comment on it so it is more obvious.

**Review by emackey — APPROVED** [review]

**Comment by scurest** [comment]
Fun fact: you can set the RGBA values above 1 in Blender. I did not know that.

**Comment by julienduroure** [comment]
Thanks!

**Pull request merged**
